tasks/tests/test_filters.py

```diff
@@ -64,3 +65,83 @@ def test_normalise_prefixed_ticket_ids(
     )
 
     assert normalised_sentence == expected_result
+
+
+@pytest.mark.parametrize(
+    ("workflow_fixture", "assignment_status", "expected_filtered_count"),
+    [
+        (["assigned_task_workflow"], "not_assigned", 0),
+        (["task_workflow"], "assigned", 0),
+        (
+            ["task_workflow", "not_assigned_task_with_previous_assignee"],
+            "not_assigned",
+            2,
+        ),
+        (
+            ["assigned_task_workflow", "not_assigned_task_with_previous_assignee"],
+            "assigned",
+            1,
+        ),
+        (
+            ["assigned_task_workflow", "not_assigned_task_with_previous_assignee"],
+            "assigned",
+            1,
+        ),
+        (
+            [
+                "assigned_task_with_previous_assignee",
+                "not_assigned_task_with_previous_assignee",
+            ],
+            "not_assigned",
+            1,
+        ),
+    ],
+    ids=["id1", "id2", "id3", "id4", "id5", "id6"],
+)
+def test_filter_by_assignment_status_workflow_list_view(
```

Comment:
Would `test_filter_by_assignment_status()` suffice as a test name since the filter is being tested in isolation without reference to the view?
```suggestion
def test_filter_by_assignment_status(
```

Reply:
Yes, this is a better name!